pkg/mixin/remote-mixins/index.json

```diff
@@ -0,0 +1,79 @@
+[
+  {
+    "name": "arm",
+    "author": "Porter Authors",
+    "description": "A mixin for deploying ARM templates on Azure",
+    "sourceURL": "https://cdn.porter.sh/mixins/arm",
```

Comment:
This should match the format used by the install commands so that someone would know how to invoke an install command based on what they found from the search results. Source URL (--url) and Feed URL (--feed-url) are mutually exclusive, only one should be set. So for our official mixins, it's the feed url.